app/components/subscription_status_component.html.erb

```diff
@@ -0,0 +1,3 @@
+<%- unless status == :success %>
```

Comment:
I think that using `render?` in the ruby file is a cleaner way to do this: https://viewcomponent.org/guide/conditional_rendering.html